Makefile

```diff
@@ -316,18 +316,14 @@ stop-kind:
 	$(ECHO)"Stopping the kind cluster"
 	$(VECHO)kind delete cluster
 
-.PHONY: install-openshift-routes
-install-openshift-routes:
-	./hack/install/install-openshift-routes.sh
```

Comment:
Would it make sense to remove this file?

Reply:
I was thinking about it, but maybe it makes sense for local development/testing without crc

Reply:
In that case, maybe, it would make sense to keep the makefile rule